sdk/main/src/impl/OpenSSLUtils.cc

```diff
@@ -17,14 +17,17 @@
  * limitations under the License.
  *
  */
-#include "impl/OpenSSLObjectWrapper.h"
+#include "impl/OpenSSLUtils.h"
 #include "exceptions/OpenSSLException.h"
 #include "impl/HexConverter.h"
-#include "impl/OpenSSLHasher.h"
 
```

Comment:
Something to consider with this class - it might be worth removing the `OpenSSL` prefix from classes and method names, e.g. `OpenSSLUtils::getErrorMessage` instead of `OpenSSLUtils::getOpenSSLErrorMessage`, `EVP_PKEY` instead of `OpenSSL_EVP_PKEY`, etc.

Since you always fully qualify these when using them anyway, it could really reduce the quantity of text needed to use these objects, without sacrificing clarity. Just a thought

Reply:
Very true, I'll remove the extra `OpenSSL` prefix

Reply:
Fixed